src/util/ringdelaybuffer.cpp

```diff
@@ -76,8 +76,8 @@ RingDelayBuffer::RingDelayBuffer(SINT bufferSize)
         : m_firstInputChunk(true),
           m_writePos(0),
           m_buffer(bufferSize) {
-    // Set the ring buffer items to 0.
-    m_buffer.fill(0);
+    // Set the ring buffer items to zero.
+    m_buffer.fill(0.0f);
```

Comment:
this does not change anything semantically its just a syntax change.

Reply:
Okay, thank you. So, based on that the PR is not necessary and can be closed.